### PR TITLE
feat: reduced >pay fee for vip users

### PR DESCRIPTION
**Pull request by Rexogamer**
currently I've set it to a 15% fee (vs 27% for normal users). if this is okay I'll make the required changes to our advertising, if not let me know and I'll tweak it. until the rate is certain I'll keep this marked as a draft.

### CODE REVIEW

File: Commands/Public/pay.js [hunk 1]
```diff
@@ -121,16 +122,16 @@ module.exports = async(client, msg, suffix) => {
 				name: `${msg.author.tag} (${msg.author.id})`,
 				icon_url: msg.author.displayAvatarURL(),
 			},
-			title: "You received money!",
-			description: "Someone gave money to you. Here's your receipt.",
+			title: "You've received money!",
+			description: "Someone's sent you some credits. Here's your receipt.",
```
Comment (turret-dev) [inline Commands/Public/pay.js:126]: someone's?

**Review by turret-dev — CHANGES_REQUESTED** [review]
seems like a typo at line 126

**Comment by Mitchell3514** [comment]
Who exactly decided this would be done at all?

**Comment by Rexogamer** [comment]
- it's more a suggestion rn
- I thought it would be a useful VIP benefit as the fee is kinda weird/high
- I submitted the PR to have the code there for reference/hopeful merger

File: Commands/Public/pay.js (hunk 1 above)
Comment (Rexogamer) [reply Commands/Public/pay.js:126]: Yes...?
73AE2DFE-944D-47F8-9E3E-D57ABD404D7D

**Comment by Mitchell3514** [comment]
Yeah current fee is derived from old Discoin transfer penalty. I've wanted to lower it for a while. Could do like 10% and 5% or even 0% for vip.

**Comment by Rexogamer** [comment]
suggestion
- 15 for normal
- 5/maybe 0 for vip

**Comment by Rexogamer** [comment]
update for external followers/for future ref: we agreed on 15/5, at this point it's just docs

**Review by Mitchell3514 — APPROVED** [review]
Fixed myself

**Comment by Mitchell3514** [comment]
Wait for codacy then can mere

File: Commands/Public/pay.js [hunk 2]
```diff
@@ -28,7 +28,8 @@ module.exports = async(client, msg, suffix) => {
 	}
 
 	// Determine the fee
-	let fee = Math.round(amount - (amount * config.transferRate));
+	let rate = fromAccount.vip ? config.vipTransferRate : config.normalTransferRate;
```
[the remark this answers is not in the document]
Comment (Rexogamer) [reply Commands/Public/pay.js]: didn't know about <user>.donator but makes sense

**Comment by Rexogamer** [comment]
Codacy seems to dislike the parentheses around your expression but 👍

**Pull request merged**